lib/iris/__init__.py

```diff
@@ -223,12 +223,12 @@ def context(self, **kwargs):
         For example::
 
             with iris.FUTURE.context():
-                iris.FUTURE.cell_datetime_objects = True
+                iris.FUTURE.cell_datetime_objects = False
```

Comment:
@LukeC92 This change is needed, but you will also have to change the comment below it.

Where it says '...code which expects time objects', this will need to be changed to '...code which expects integers' or whatever the current default is (you will have to check this by trying it or looking at the Iris docs).

This is because that comment is now describing _past_ behaviour, not _future_ behaviour.